dart-impl/mpi/src/dart_mem.c

```diff
@@ -39,11 +39,18 @@ struct dart_buddy {
 char* dart_mempool_localalloc;
 struct dart_buddy  *  dart_localpool;
 
-static inline int
+static inline unsigned int
```

Comment:
Why not make this `size_t`?

Reply:
Because it is only going to store values [0 - 63], and it the returned value of this function is stored in an unsigned integer that is part of a data structure, that is probably used in some other place... not sure what would be the consequences.